apps/ui/src/components/Ui/Markdown.vue

```diff
@@ -1,21 +1,48 @@
 <script setup lang="ts">
+import hljs from 'highlight.js/lib/core';
+import javascript from 'highlight.js/lib/languages/javascript';
+import json from 'highlight.js/lib/languages/json';
+import rust from 'highlight.js/lib/languages/rust';
+import python from 'highlight.js/lib/languages/python';
 import { computed } from 'vue';
 import { Remarkable } from 'remarkable';
+import { icons } from '@iconify-json/heroicons-outline';
 import { linkify } from 'remarkable/linkify';
 import { getUrl } from '@/helpers/utils';
 
+hljs.registerLanguage('javascript', javascript);
+hljs.registerLanguage('json', json);
+hljs.registerLanguage('rust', rust);
+hljs.registerLanguage('python', python);
```

Comment:
Does highlight.js support Solidity? Solidity was suggested by less and it's missing there.

Reply:
It does not, it requires this plugin https://github.com/highlightjs/highlightjs-solidity

Should we proceed with this additional dependency ?

Reply:
Looks like it works with older version of highlight (v9).

Reply:
v9 is pre-covid era

Reply:
Let's skip Solidity for now, not a big deal

Reply:
It will fallback and be parsed as javascript I think